plugins/attachment/src/index.ts

```diff
@@ -33,6 +33,9 @@ export interface Attachment extends AttachedDoc {
   pinned?: boolean // If defined and true, will be shown in top of attachments collection
 
   readonly?: boolean // If readonly, user will not be able to remove or modify this attachment
+  originalWidth?: number
```

Comment:
Do we want to put all attributes into the same Interface? Maybe we can add one field that contains all additional metadata for this type of attachment? For example:

```
export interface ImageMetadata {
    originalWidth: number
    originalHeight: number
    pixelRatio: number
}
export type AttachmentMetadata = ImageMetadata | VideoMetadata | ...
...
metadata?: AttachmentMetadata
```

Reply:
yes that makes sense, thanks
Done